src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -865,6 +867,77 @@ public boolean onRightClick(ItemUseEvent e, Player p, ItemStack item) {
 		.register(true);
+		
+		new SlimefunItem(Categories.MAGIC, SlimefunItems.STAFF_STORM, "STAFF_ELEMENTAL_STORM", RecipeType.ANCIENT_ALTAR,
+		new ItemStack[] {SlimefunItems.RUNE_AIR, SlimefunItems.RUNE_WATER, SlimefunItems.ENDER_LUMP_3, SlimefunItems.STAFF_WATER, SlimefunItems.MAGIC_SUGAR, SlimefunItems.STAFF_WIND, SlimefunItems.ENDER_LUMP_3, SlimefunItems.RUNE_WATER, SlimefunItems.RUNE_AIR})
+		.register(true, new ItemInteractionHandler() {
+
+			@Override
+			public boolean onRightClick(ItemUseEvent e, Player p, ItemStack item) {
+				//Not checking if lores equals because we need a special one for that.
+				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.STAFF_STORM, false)) {
+
+					if (!item.hasItemMeta()) return false;
+					ItemMeta itemM = item.getItemMeta();
+					if (!itemM.hasLore()) return false;
+					List<String> itemML = itemM.getLore();
+
+					ItemStack SFitem = SlimefunItems.STAFF_STORM;
+					ItemMeta SFitemM = SFitem.getItemMeta();
+					List<String> SFitemML = SFitemM.getLore();
+					if (itemML.size() < 6) {
+						// Index 1 and 3 in SlimefunItems.STAFF_STORM has lores with words and stuff so we check for them.
+						if (itemML.get(1).equals(SFitemML.get(1)) && itemML.get(3).equals(SFitemML.get(3))) {
+							if (p.getFoodLevel() >= 4 || p.getGameMode() == GameMode.CREATIVE) {
+								Location loc = p.getTargetBlock(null, 50).getLocation();
+								if (loc.getWorld() != null && loc.getChunk().isLoaded()) {
+									if (new ProtectionManager(Bukkit.getServer()).hasPermission(p, loc, ProtectionModule.Action.PVP)) {
+										loc.getWorld().strikeLightning(loc);
+
+										if (p.getInventory().getItemInMainHand().getType() != Material.SHEARS && p.getGameMode() != GameMode.CREATIVE) {
+											FoodLevelChangeEvent event = new FoodLevelChangeEvent(p, p.getFoodLevel() - 4);
+											Bukkit.getPluginManager().callEvent(event);
+											p.setFoodLevel(event.getFoodLevel());
+										}
+
+										if (ChatColor.translateAlternateColorCodes('&', "&e5 Uses &7left").equals(itemML.get(4))) {
+											itemML.set(4, ChatColor.translateAlternateColorCodes('&', "&e4 Uses &7left"));
+										} else if (ChatColor.translateAlternateColorCodes('&', "&e4 Uses &7left").equals(itemML.get(4))) {
+											itemML.set(4, ChatColor.translateAlternateColorCodes('&', "&e3 Uses &7left"));
+										} else if (ChatColor.translateAlternateColorCodes('&', "&e3 Uses &7left").equals(itemML.get(4))) {
+											itemML.set(4, ChatColor.translateAlternateColorCodes('&', "&e2 Uses &7left"));
+										} else if (ChatColor.translateAlternateColorCodes('&', "&e2 Uses &7left").equals(itemML.get(4))) {
+											itemML.set(4, ChatColor.translateAlternateColorCodes('&', "&e1 Use &7left"));
+										} else if (ChatColor.translateAlternateColorCodes('&', "&e1 Use &7left").equals(itemML.get(4))) {

```

Comment:
Nitpicky, but you should maybe put this inside a for-loop, that way you can also easily bump up the uses later on.
For when a Lightning Rune is added and the recipe is even more complex that way, then the uses could be slightly more too.

Reply:
Can't think how can I made a for-loop in this scenario

Reply:
Then I will do that one.